### ISSUE TITLE
IndexError: list index out of range

### ISSUE THREAD

**Issue by Haider8** [roshnet/echelon-transform#3]
After taking the input of number of rows and column the following error occurs after clicking "process data" button:
**IndexError: list index out of range**

**Comment by roshnet** [comment]
@haider8 Yes there's this flaw in the application, which needs to be fixed.
But, unlike you sound, the flaw is in the front-end part, not the Python code.
### How to get it running despite the flaw
If you want to test out the application, do this:
- When asked for the dimensions, refresh the page.
- Run it again.
### Possible solution
Maybe the JS needs to be asynchronous. That way, it will always wait for the user to enter the values, and then present the required number of fields.
PS: refreshing the page also does the same thing.

Hoping for contributions. :+1:
